code/vocabulary.py

```diff
@@ -1,3 +1,5 @@
+from typing import Dict, Union
```

Comment:
looks like `Dict` isn't used

Reply:
It's used in a type annotation of `PhraseReplacer.__init__`, https://github.com/JelleAalbers/knausj_talon/blob/replace_phrases/code/vocabulary.py#L113; unless I'm missing something